content/_partners/faq._en.md

```diff
@@ -73,59 +69,33 @@ logistics_accordion:
+            <p>Non-U.S. citizens and non-U.S. immigrants can authenticate with Login.gov, though select features (e.g., SMS / voice OTP for MFA) may be restricted in certain countries. Check our [International phone number support](/help/manage-your-account/international-phone-support/){:class="external-link"} for a complete list that Login.gov supports for authenticating end-users.</p>
+            
+            <p>Non-U.S. citizens and non-U.S. immigrants can verify their identity (i.e. “proof”) with Login.gov as long as they have a valid U.S. state-issued ID, social security number (SSN), and U.S. address.</p>
     -
         id: b-6
-        title: What is identity proofing?
-        content: >-
-            “Identity proofing is the process by which a [credentialing service provider] collects, validates, and verifies information about a person.” (NIST) This is the process Login.gov uses to verify that a user is who they say they are. While many agencies can validate an individual’s identity through an in-person proofing experience, we developed an online application that allows individuals to have their identities verified from their smartphone or computer.
-    -
-        id: b-7
-        title: What is the proofing rate?
-        content: >-
-            The proofing rates will vary by population and we work with the partner to find a solution for users who are not able to proof online. The goal is to provide a high level of assurance. Since launch, we have seen 65%-70% of users who attempt to upload an ID will proof their identity. This includes transactions classified as high risk and likely (or potentially) fraudulent.
-    -
-        id: b-8
-        title: Can Login.gov authenticate or proof non-U.S. citizens or non-U.S. immigrants (not a U.S. citizen, U.S. national, lawful permanent resident, or traveling to the United States on an immigrant visa)?
-        content: >-
-            Check [International phone number support](/help/manage-your-account/international-phone-support/){:class="external-link"} for a complete list that Login.gov supports for authenticating end-users. In addition to a phone number, at this time Login.gov requires users to proof with a state-issued ID card and social security number.
-    -
-        id: b-9
         title: What forms of identification can Login.gov accept for identity proofing?
         content: >-
             At this time, only the following state-issued identification is accepted: 
     
-            - Driver’s license from all 50 states and other US territories (Guam, US Virgin Islands, Mariana Islands and Puerto Rico)
+            - Driver’s license from all 50 states, the District of Columbia (DC), and other US territories (Guam, US Virgin Islands, Mariana Islands and Puerto Rico)
 
             - A non-driver’s license state-issued ID card
                 – This is an identity document issued by the state/US territory that asserts identity but does not give driving privileges.
```

Comment:
Does making this a double-dash allow this to render as a sub-bullet instead of a dash? That might look better (screenshot to show what it looks like now).
<img width="778" alt="Screen Shot 2022-08-04 at 3 11 38 PM" src="https://user-images.githubusercontent.com/51330839/182961493-2127fe2d-2fb9-4981-b907-96e6eab48bb0.png">

Reply:
This is very minor, and I don't remember what the Markdown related to nested lists are, so feel free to ignore.

Reply:
Spaces vs tabs ... tabs don't render as sub-lists but spaces do.